addons/account/models/partner.py

```diff
@@ -433,7 +433,16 @@ def _get_company_currency(self):
                 partner.currency_id = self.env.company.currency_id
 
     credit = fields.Monetary(compute='_credit_debit_get', search=_credit_search,
-        string='Total Receivable', help="Total amount this customer owes you.")
+        string='Total Receivable', help="Total amount this customer owes you.",
+        groups='account.group_account_invoice,account.group_account_readonly',
+        currency_field='currency_id')
+    credit_limit = fields.Float(string='Partner Credit Limit', company_dependent=True, copy=False)
+    credit_limit_compute = fields.Monetary(
+        string='Credit Limit', help='Credit limit specific to this partner.', currency_field='currency_id',
+        compute='_compute_credit_limit_compute', inverse='_inverse_credit_limit_compute')
+    use_partner_credit_limit = fields.Boolean(string='Partner Limit', company_dependent=True,
```

Comment:
I'm not sure I get why this field is useful.
It is used to
* compute `credit_limit_compute` but that could be done with storing the default value in `ir.property` without a `res_id`
* know if you should update the `ir.property`, why wouldn't you if you change the value?
* set the visibility of the field `credit_limit_compute`, but I think that could be done using `Model<ir.property>._get_default_property` to avoid storing values in `ir.property` twice for the same thing

Reply:
We added this field so we would not have to use "codes" like "-1" to know when the partner would no longer have a specific limit and should fall back to the default limit.
It is linked to the fact that the client automatically converts an empty field to "0", so we cannot use an empty field as an indicator to use the default limit.